_scripts/instruction-widget/templates/install/centos.html

```diff
@@ -1,6 +1,13 @@
-{{> header}}
-
+{{#deprecated_32bits}}
+<aside class="note">
+  <div class="note-header">
+    <h3>Supported architectures:</h3>
+  </div>
+  <p>We only support CentOS/RHEL 6 systems running on x86_64 architecture. Pouet.</p>
```

Comment:
What about:

> Certbot only supports CentOS/RHEL 6 systems running on the x86_64 architecture. To use Certbot on another architecture, you will need to upgrade your OS.

I also think it could be nice to make CentOS/RHEL 6 dynamic and say the name of the selected OS, but I don't really care if you leave that out.

Reply:
As a general deprecation notice, I do not think it alters the purpose to keep both names each time. So I will be more in favor of the most simple implementation, that is to keep the message static.

Reply:
That's fine. I'd still like the text to be changed to something like what I suggested though.

Reply:
Done.